app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/view/message/PProUpsellBannerPlugin.kt

```diff
@@ -73,7 +70,7 @@ class PProUpsellBannerPlugin @Inject constructor(
                         Message(
                             topIllustration = com.duckduckgo.mobile.android.R.drawable.ic_privacy_pro,
                             title = context.getString(R.string.apptp_PproUpsellBannerTitle),
-                            subtitle = subtitle,
+                            subtitle = context.getString(R.string.apptp_PproUpsellBannerMessage_freeTrial),
```

Comment:
I don't see any rebranding FF checks in this file, so it seems we are just simplifying the copies for this banner(?).

Related to that, we are now using as subtitle `apptp_PproUpsellBannerMessage_freeTrial` for both: a) user free trial eligible, and b) free trial not eligible. 

Can you double check is it correct to use `_freeTrial` if `if (subscriptions.isFreeTrialEligible() && appTPStateMessageToggle.freeTrialCopy().isEnabled())` returns false?

Reply:
Yes, it is correct, sorry for the confusion. 
My plan is handle all those naming inconsistencies when requesting translations, as doing it here would just get overridden and mess up smartling.